Library/ViewModels/PledgePaymentMethodsViewModel.swift

```diff
@@ -5,10 +5,6 @@ import ReactiveSwift
 import UIKit
 
 public typealias PledgePaymentMethodsValue = (user: User, project: Project, applePayCapable: Bool)
-public typealias CardViewValues = (
```

Comment:
This type seemed very similar to `PledgeCreditCardValue`, removed this one and renamed that to `PledgeCreditCardViewData`.